osama-breakout-room/Shirin Muataz/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="Style.css" />
+    <script src="https://kit.fontawesome.com/9016f6ba0a.js"></script>
+  </head>
+
+  <body>
+    <header>
+      <div class="wrapper">
+        <div class="logo">
+          <img src="./images/logo.png" alt="logo" />
+        </div>
+
+        <nav>
+          <a class="btn btn-bordered" href="#">
+            LOGIN
+          </a>
+          <a class="btn btn-blue" href="#">
+            REGISTER NEW ACCOUNT
+          </a>
+        </nav>
+      </div>
+    </header>
+
+    <div class="wrapper">
+      <main>
```

Comment:
Awesome job on using Semantic HTML. 👍 It makes reading and maintaining your code a lot easier.